packages/itwinui-css/src/text/muted.scss

```diff
@@ -1,5 +1,5 @@
 // Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @mixin iui-text-muted {
-  color: var(--iui-color-foreground-4);
+  color: var(--iui-color-foreground-3);
```

Comment:
Changed from 4 to 3 and changed the lightness of the foreground colors.  Confirmed passing in all themes on `color.html`.